packages/pharos/src/components/modal/pharos-modal.ts

```diff
@@ -211,6 +211,19 @@ export class PharosModal extends LitElement {
     }
   }
 
+  private _emitEvent() {
```

Comment:
Could this name be scoped a bit better? e.g. `_emitVisibilityChange` or something of the sort.